development.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.4"
+services:
+  beacon_node:
+    build:
+      target: development
+    image: lodestar:dev
+    volumes:
+      - ./:/usr/app
+      - ../dev/lodestar/node_modules:/usr/app/node_modules
+    command: tail -F anything
```

Comment:
Why this command?

Reply:
this is to keep the container spinning, so as to provide shells for the developer to run installs, beacon and/or val or dev/test individual segments, this overrides the main docker-compose command so the image doesn't start executing default code.